template/studio/resolveDocumentActions.js

```diff
@@ -1,3 +1,13 @@
+/*
```

Comment:
Document actions will enter the template when I include the code from https://demo-custom-workflow.sanity.studio/workflow. So all of this can really be removed.